src/pages/[platform]/build-a-backend/auth/use-existing-cognito-resources/index.mdx

```diff
@@ -112,12 +112,60 @@ Configuring the mobile client libraries directly is not supported, however you c
 
 ## Use auth resources with an Amplify backend
 
-<Callout warning>
+If you have created Amazon Cognito resources outside of the context of your Amplify app such as creating resources through the AWS Console or consuming resources created by a separate team, you can use `referenceAuth` to reference the existing resources.
```

Comment:
can we add guidance here on what is supported? ie: ```a user pool and identity pool with auth and unauth roles, as well as a user pool client``` Perhaps it is assumed that their auth structure should be typical of an Amplify app